eng/targets/VisualStudio.targets

```diff
@@ -143,4 +143,32 @@
   </Target>
 
   <Import Project="GeneratePkgDef.targets" Condition="'$(MSBuildRuntimeType)' != 'Core' and '$(GeneratePkgDefFile)' == 'true' and '$(DesignTimeBuild)' != 'true' and '$(BuildingForLiveUnitTesting)' != 'true'" />
+
+  <!--
+    Workaround for missing dependency publishing feature in the VS SDK.
+    
+    Relies on the following convention: When a project reference publish outputs (i.e. more than standard build outputs) need to be packaged into a VSIX,
+    the project should define VsixItemsOutputGroup target and include it in ProjectRefrence.IncludeOutputGroupsInVSIX.
```

Comment:
I'm not a fan of the name `VsixItemsOutputGroup` here, because the actual VSIX is produced by the Build step and not the Publish step, so it's confusing to use the same name to refer specifically to Publish outputs.

Reply:
Suggestions for a better name? `VsixPublishedItemsOutputGroup`?

Reply:
I guess `PublishedProjectOutputGroup` would be the closest match to `BuiltProjectOutputGroup`, unless you only intended to include a subset of the published items in the VSIX. Any of these is fine though.